extensions/wikia/Chat2/ChatBanListSpecial_helper.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+/**
+ * @package MediaWiki
+ * @subpackage SpecialPage
+ * @author Piotr Molski <moli@wikia.com>
+ * @version: $Id$
+ */
+
+use Wikia\Service\User\Permissions\PermissionsServiceAccessor;
+
+class ChatBanData {
+	use PermissionsServiceAccessor;
+	
+	private $mCityId;
+	private $mUserName;
+	private $mLimit;
+	private $mOffset;
+	private $mOrder;
+	private $mOrderOptions;
+    private $sk;
+	private $mDBh;
+	private $mTable;
+
+    function __construct ($city_id, $load = 1) {
+        global $wgExternalDatawareDB;
+
+        $this->mCityId = $city_id;
+        $this->mDBh = $wgExternalDatawareDB;
+        $this->mTable = 'chat_ban_users';
+        $this->sk = RequestContext::getMain()->getSkin();
+
+        $this->mOrderOptions = [
+            'timestamp' => ['start_date %s'],
+            'target'    => ['cbu_user_id %s'],
+            'expires'   => ['end_date %s'],
+            'blockedBy' => ['cbu_admin_user_id %s'],
+            'reason'    => ['reason %s'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "reason" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)